### ISSUE TITLE
Is there a way to disable the automatic pop up and only use the force hash?

### ISSUE THREAD

**Issue by HandHugs** [tylerpearson/CoverPop#28]
I have a project where we dont necessarily want the pop up to appear automatically, but we do want to make use of the forceHash function. Is that possible?

**Comment by samford** [comment]
I just happened to look at this project for the first time in a while, so maybe I can help.

Looking at the code for the `CoverPop.init` function, the popover will be displayed if A) the cookie isn't present or B) the hash corresponding to the `settings.forceHash` property is present (in `window.location`).  With that in mind, to get the desired behavior you could either manually set the cookie before you call `CoverPop.init`/`CoverPop.start` or you could modify CoverPop to have an extra setting that effectively prevents `CoverPop.open` from being called on init.


For the cookie method, you could do something like this (I'm recycling the code from the `CoverPop.util.setCookie` function below since it's not publicly accessible).

```javascript
// Define the CoverPop settings
var coverPopSettings = {
  coverId:             'CoverPop-cover',
  expires:             30,
  closeClassNoDefault: 'CoverPop-close',
  closeClassDefault:   'CoverPop-close-go',
  openClassDefault:    'CoverPop-open',
  cookieName:          '_CoverPop',
  onPopUpOpen:         function() {},
  onPopUpClose:        function() {},
  forceHash:           'splash',
  delayHash:           'go',
  closeOnEscape:       true,
  delay:               0,
  hideAfter:           null
};

// Set the CoverPop cookie
var date = new Date();
date.setTime(+ date + (coverPopSettings.expires * 86400000));
document.cookie = coverPopSettings.cookieName + '=true; expires='
  + date.toGMTString() + '; path=/';

// Initialize CoverPop
CoverPop.start(coverPopSettings);
```

Since the cookie is set before CoverPop is initialized, `CoverPop.open` isn't executed in `CoverPop.init`.  `forceHash` overrides the cookie, so you'll still be able to use that on demand.


If the cookie method won't work for your particular situation, you can easily modify CoverPop to have a setting that prevents CoverPop from opening on init.  For this method, you can refer to the open-on-init-setting branch of my fork: [samford/CoverPop/tree/open-on-init-setting](https://github.com/samford/CoverPop/tree/open-on-init-setting)

Basically, the `openOnInit` setting defaults to true (mimicking CoverPop's current behavior) but if you set it to false it won't display the popover on init like it normally would (if the cookie isn't set).


All that having been said, I would probably go with the cookie method if possible, as it doesn't require any modifications to the CoverPop code.  However, if you're into the idea of the `openOnInit` setting, I can create a pull request (though it could take a while for it to get merged).

Anyway, I hope this is helpful!
